<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>611</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane
{
	border: -1px;
	top:-2px;
	left: 1px;
}
 
QTabBar::tab:first		/*第一个页面上面的标签tab*/
{
	color:#333333;
	background:transparent;
	
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	
	padding-left:-9px;
	padding-right:-9px;
	width:105px;
	height:30px;
	margin-left:0px;
	margin-right:40px;
}
 
QTabBar::tab:last		/*最后一个页面上面的标签tab*/
{
	color:#333333;
	background:transparent;
	
	font-family:&quot;微软雅黑&quot;;
	font-size:14px;
	
	padding-left:-9px;
	padding-right:-9px;
	width:75px;
	height:30px;
	margin-left:0px;
	margin-right:40px;
}
 
QTabBar::tab:selected, QTabBar::tab:hover
{
	color:#618BE5;
	background:transparent;
	
	font-family:&quot;微软雅黑&quot;;	
}

 
 
#detailWgt, #appealReasonWgt	/*设置下面的单个widget没有边框*/
{
	border:none;
}

QListWidget::Item{padding-left:0px;padding-top:20px; padding-bottom:4px;color:white}
QListWidget::Item:hover{background:lightgray; color:white;}
QListWidget::item:selected{background:lightgray; color:white; }
QListWidget{border: 1px solid black;}
QListWidget{border-radius: 10px;}
QListWidget{padding: 0 8px;}
QListWidget{background-color: rgb(255, 255, 255);}
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="onePage">
       <attribute name="title">
        <string>单文件模式</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,6,0">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="horizontalSpacing">
         <number>16</number>
        </property>
        <property name="verticalSpacing">
         <number>35</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="QWidget" name="widget_3" native="true"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,3,3">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>输入：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="imgPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSelect">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>选择照片</string>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/img/img/select.png</normaloff>:/img/img/select.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonStart">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>开始</string>
            </property>
            <property name="icon">
             <iconset resource="../resource.qrc">
              <normaloff>:/img/img/start.png</normaloff>:/img/img/start.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>448</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,9">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>进  度:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_3">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QWidget" name="widget_2" native="true"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(192, 192, 192);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
          </property>
          <property name="text">
           <string>提示:该模型对自然景物上色比较明显，对人物上色效果不明显 </string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QListWidget" name="listWidget">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="towPage">
       <attribute name="title">
        <string>多张图片处理</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,1,1,6,0">
        <property name="horizontalSpacing">
         <number>16</number>
        </property>
        <property name="verticalSpacing">
         <number>26</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_8" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3,3,3">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>输入：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="imgPath_4"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSelect_4">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>选择文件夹</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/img/img/select.png</normaloff>:/img/img/select.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStart_4">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>开始</string>
              </property>
              <property name="icon">
               <iconset resource="../resource.qrc">
                <normaloff>:/img/img/start.png</normaloff>:/img/img/start.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>422</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>进  度：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(192, 192, 192);
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
   </string>
          </property>
          <property name="text">
           <string>提示:该模型对自然景物上色比较明显，对人物上色效果不明显 </string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QListWidget" name="listWidget_4">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
